experimental/PropertyDDS/packages/property-shared-tree-interop/src/schemaConverter.ts

```diff
@@ -9,7 +9,7 @@ import {
 	FieldKinds,
 	TreeFieldSchema,
 	SchemaBuilderBase,
-	FieldKind,
+	OldFieldKind as FieldKind,
```

Comment:
Renamed EditableTree2 'FieldKind' to 'OldFieldKind' to address naming collision when exporting the new 'FieldKind' from "./class-tree".